core/src/main/java/org/apache/iceberg/avro/Avro.java

```diff
@@ -84,6 +84,7 @@ public static WriteBuilder write(OutputFile file) {
     private Map<String, String> config = Maps.newHashMap();
     private Map<String, String> metadata = Maps.newLinkedHashMap();
     private Function<Schema, DatumWriter<?>> createWriterFunc = GenericAvroWriter::new;
+    private boolean overwrite = true;
```

Comment:
I think this should default to `false`.

Reply:
Now all readers default to overwrite false.